modin/data_management/functions/default_methods/groupby_default.py

```diff
@@ -86,15 +83,7 @@ def fn(
                 else agg_func(grp, **agg_args)
             )
 
-            if not is_multi_by:
-                if as_index:
-                    return result
-                else:
-                    if result.index.name is None or result.index.name in result.columns:
-                        drop = False
-                    return result.reset_index(drop=not drop)
-            else:
-                return result
+            return result
```

Comment:
it seems that `BaseOnPython` CI job has failed to report code coverage, because going by codecov report, not a single line of code in `BaseQueryCompiler` was not covered by tests.

I'll rerun CI to see if it help